src/neuroconv/tools/hdmf.py

```diff
@@ -1,11 +1,14 @@
 """Collection of modifications of HDMF functions that are to be tested/used on this repo until propagation upstream."""
-from typing import Tuple
+from typing import Dict, Tuple
 
 import numpy as np
-from hdmf.data_utils import GenericDataChunkIterator as HDMFGenericDataChunkIterator
 
+# from hdmf.data_utils import GenericDataChunkIterator as HDMFGenericDataChunkIterator
+from hdmf.data_utils import GenericDataChunkIterator
 
-class GenericDataChunkIterator(HDMFGenericDataChunkIterator):
+
+# class GenericDataChunkIterator(HDMFGenericDataChunkIterator):
+class _GenericDataChunkIterator(GenericDataChunkIterator):
```

Comment:
This was a temporary reversion, I think I can restore it - just need to test it doesn't break the serialization